build/yamls/antrea-aks.yml

```diff
@@ -4130,6 +4187,44 @@ spec:
           type: DirectoryOrCreate
         name: host-var-log-antrea
 ---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: antrea
+    component: antrea-watcher
```

Comment:
I think `antrea-config-watcher` is a better name

Reply:
sure, I will on hold the update of this PR until there are more comments from @jianjuns,@tnqn and @salv-orlando in case there is a disagreement to add new deployment. thanks.